solr/docker/build.gradle

```diff
@@ -75,8 +75,7 @@ dependencies {
     builtBy 'createDockerfileOfficial'
   }
 
-  solrTgz project(path: ":solr:distribution:", configuration: 'solrTgz')
-  solrTgzSignature project(path: ":solr:distribution", configuration: 'solrTgzSignature')
```

Comment:
Doesn't this need to stay around? Where does the solrTgzSignature come from now?

Reply:
It works by magic :) 

Turns out with this new setup, we don't actually need the signed artifacts or the asc signature, we only need the gpg key fingerprint of the RM, sourced to populate `releaseGpgFingerprint`, and that is fetched from property `signing.gnupg.keyName` for gnuPg or `signing.keyId` for java-version.

So even if we don't produce the final distribution with signatures, the `gradlew createDockerfileOfficial` works as long as you have set `-Psigning.keyId=xxxxxx` :) Nice!

Reply:
ahhh we don't need it for the createDockerfileOfficial, but I do think we need it for `testDockerfileOfficial`, because that builds the dockerfile with a local server that hosts the tarball and asc signature.

Will pull this locally and try things out, and tinker around.

Reply:
You may be right. The tests have not ben happy on my Mac so I have not really tried to run them through gradle. Is there a way to run the `testDocker` task and have it use bash and gnu tools on my mac (similar to init-macos script in docker-solr)`

Reply:
I try running `testBuildDockerfileOfficial` but the MockServer thing is not working on  my mac...